app/secrets.py

```diff
@@ -1,6 +1,6 @@
 from jwcrypto import jwk
```

Comment:
All of this logic has also been replaced in the new library. or are we blocked from doing this until the secrets and keys have been split?

Reply:
yeah this is blocked until we split the keys from secrets. Card in ready for dev currently